bootstrap-salt.sh

```diff
@@ -1463,7 +1468,7 @@ fi
 
 # Starting from Ubuntu 16.10, gnupg-curl has been renamed to gnupg1-curl.
 GNUPG_CURL="gnupg-curl"
-if ([ "${DISTRO_NAME_L}" = "ubuntu" ] && [ "${DISTRO_VERSION}" = "16.10" ]); then
+if ([ "${DISTRO_NAME_L}" = "ubuntu" ] && ([ "${DISTRO_VERSION}" = "16.10" ] || [ "$DISTRO_MAJOR_VERSION" -gt 16 ])); then
```

Comment:
Just for the sake of prettiness, it would be good to split this long statement in two lines :smiley: